content/platform/data-services-cloud-console-api/home.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Data Services Cloud Console NorthBound API
+version: 0.1.1
+description: Easy to read guides, tips and documentation to rapidly help any
+  developer working with Data Services Cloud Console APIs.
+image: /img/platforms/image-1-dscc.svg
+width: large
+priority: 1
+tags:
+  - data-services-cloud-console
+  - greenlake-common-cloud
+---
+## Intro
```

Comment:
I suggest getting rid of Intro, as we don;t have it in any other platform page.

Reply:
Done!!